vllm/sampling_params.py

```diff
@@ -184,8 +184,10 @@ class SamplingParams(
     ignore_eos: bool = False
     max_tokens: Optional[int] = 16
     min_tokens: int = 0
-    logprobs: Optional[int] = None
-    prompt_logprobs: Optional[int] = None
+    # Number of sample logprobs and prompt logprobs,
+    # respectively, requested
+    request_sample_logprobs: Optional[int] = None
+    request_prompt_logprobs: Optional[int] = None
```

Comment:
Is this change necessary? `SamplingParams` is an end-user interface for the `LLM` API, so its API shouldn't change.